dataactvalidator/config/sqlrules/b11_award_financial_3.sql

```diff
@@ -3,4 +3,5 @@ SELECT
     af.object_class
 FROM award_financial AS af
 WHERE af.submission_id = {}
-AND af.object_class = '000'
+AND LENGTH(af.object_class) <> 3
+AND af.object_class <> '0'
```

Comment:
Should never be 0 because of our padding. We can also probably just change this back since we still need it to be a warning.